johnsonandjohnson/Bodiless-JS
# fix(test-site, documentation): Fixed a image location in documentation and other small fixes

## Changes

* Moved tout image to the correct location in the documentation for it to load properly
* Change the styling on the homepage list so that it can have items added to it and keep them all on one line
* change the default template for new pages to have the default flexbox on it.

## Test Instructions


Go to the documentation page Design System
See at the very bottom the Tout token flow

On the Home page add a new item in the list and see it becomes a 4 by 4 list.

Add new page from the home page 
get a page that has header flexbox and footer

## Related Issues
